spec/nine_one_one/slack_service_spec.rb

```diff
@@ -1,13 +1,13 @@
 require 'spec_helper'
 
-describe NineOneOne::SlackService do
-  describe '#notify' do
-    let(:webhook_url) { 'https://hooks.slack.com/services/XXX' }
+RSpec.describe NineOneOne::SlackService do
+  let(:webhook_url) { 'https://hooks.slack.com/services/XXX' }
 
+  describe 'with no optional fields' do
     subject { described_class.new(webhook_url) }
 
     describe 'when API call succeeds' do
-      use_vcr_cassette 'slack_notification_success', match_requests_on: [:body]
+      use_vcr_cassette 'slack_notification_success'
```

Comment:
I think that `match_requests_on: [:body]` enables body check. So by disabling it you stop checking if the request body was as expected. Am I right?